### DISCUSSION TITLE
node-compat mode doesn't include process polyfill

### DISCUSSION THREAD

**Discussion by anta-semenov** [cloudflare/workers-sdk#2897] [Q&A]
Hello.

Is there a reason why node-compat mode doesn't enable `process` polyfill? 

https://github.com/cloudflare/workers-sdk/blob/f61de1dd1b09cb45fefb6e2ace2e615f6a87ef4b/packages/wrangler/src/bundle.ts#L379

**Comment by JacobMGEvans** [comment]
The `nodejs_compat` is currently a very limited and undermaintained set of polyfills from the ESBuild ecosystem, there is current work internally implementing more robust and direct support for Node in the Workers Runtime, through compatibility flags you can see it here: `nodejs_compat`
https://miniflare.dev/core/compatibility
